src/BuildIt.Forms/BuildIt.Forms.Controls/CameraPreviewControl.xaml.cs

```diff
@@ -307,10 +298,45 @@ internal async Task OnMediaFrameArrived(MediaFrame mediaFrame)
             await MediaFrameArrived(mediaFrame);
         }
 
+        /// <summary>
+        /// Raises an error when something wrong happened when opening a camera
+        /// </summary>
         internal void RaiseErrorOpeningCamera()
         {
             Status = CameraStatus.Stopped;
             ErrorOpeningCamera?.Invoke(this, EventArgs.Empty);
         }
+
+        private static void FocusModePropertyChanging(BindableObject bindable, object oldvalue, object newvalue)
```

Comment:
Is there a reason why we have separate methods for changing and changed?  It doesn't look like changing prevents the change, or sets it back to what it was, which is what I'd expect it to do if there was an error